help/tquery.md

```diff
@@ -0,0 +1,52 @@
+# Tquery
+
+### Typy danych i operatory filtrów
+
+| typ          | null(5) | = | in   | cmp | like | select | sort |
+|--------------|---------|---|------|-----|------|--------|------|
+| bool         | *       | * | .    | .   | .    | *      | +    |
+| date         | *       | * | *    | *   | .    | *      | +    |
+| datetime     | *       | . | .    | *   | .    | *      | +    |
+| int          | *       | * | *    | *   | *    | *      | +    |
+| string       | *       | * | *    | *   | *    | *      | +    |
+| uuid         | *       | * | *    | .   | .    | *      | .    |
```

Comment:
Weź tę linię przesuń w dół o jeden. Po pierwsze będzie alfabetycznie, po drugie text będzie przy string co też ma pewien sens.